src/value.rs

```diff
@@ -48,6 +48,7 @@ pub enum Value {
     Float(Option<f32>),
     Double(Option<f64>),
     String(Option<Box<String>>),
+    Char(Option<char>),
```

Comment:
Actually, I think we missed some impl in various drivers?

Reply:
I think we can remove this. This is my mistake...